cli/src/stake.rs

```diff
@@ -684,12 +684,18 @@ impl StakeSubCommands for App<'_, '_> {
                         .takes_value(false)
                         .help("Display inflation rewards"),
                 )
+                .arg(
+                    Arg::with_name("csv")
+                        .long("csv")
+                        .takes_value(false)
+                        .help("Format stake account data in csv")
+                )
                 .arg(
                     Arg::with_name("num_rewards_epochs")
                         .long("num-rewards-epochs")
                         .takes_value(true)
                         .value_name("NUM")
-                        .validator(|s| is_within_range(s, 1..=10))
+                        .validator(|s| is_within_range(s, 1..=50))
```

Comment:
this isn't related to the csv format change (that's fine, just making a comment for additional awareness).

Why was this previously 10, but can now be 50? what is the reason this is limited at all?

Reply:
Wow, I was surprised to see that running this with a value of 50 does actually complete (after 8+ min). I feel ambivalent about changing this.

Reply:
I'm not sure why it's 10, but that's quite limiting. 50 does work but it's slow. It should definitely be larger than 10 or maybe have a way to have an offset to older values.